test/extended/registry/signature.go

```diff
@@ -0,0 +1,120 @@
+package registry
```

Comment:
IMHO this should belong rather to `test/extended/imageapis`. But I admit it's ambiguous. Tests in the `imageapis` directories work with registry as well. And the same goes for tests in `images` and `builds` directory.